### PR TITLE
Callback examples

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

**Review by ihar-kahadouski — APPROVED** [review]

File: example-testng-logback/src/main/resources/reportportal.properties [hunk 1]
```diff
@@ -1,6 +1,7 @@
-rp.endpoint=https://beta.demo.reportportal.io
-rp.uuid=51aabf1d-958c-4f14-92b5-00380884c3c5
+rp.endpoint=https://example.com
+rp.uuid=uuid
```
Comment (HardNorth) [inline example-testng-logback/src/main/resources/reportportal.properties]: Please revert this change

File: example-testng-logback/pom.xml [hunk 2]
```diff
@@ -21,10 +21,10 @@
     <dependencies>
 
         <dependency>
-            <groupId>com.epam.reportportal</groupId>
-            <!--<groupId>com.github.reportportal</groupId>-->
+<!--            <groupId>com.epam.reportportal</groupId>-->
+            <groupId>com.github.reportportal</groupId>
             <artifactId>agent-java-testng</artifactId>
-            <version>5.0.0-BETA-6</version>
+            <version>895e2b28</version>
```
Comment (HardNorth) [inline example-testng-logback/pom.xml]: Guess that too outdated, please update dependencies.

File: example-junit5/pom.xml [hunk 3]
```diff
@@ -14,9 +14,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.epam.reportportal</groupId>
+            <groupId>com.github.reportportal</groupId>
             <artifactId>agent-java-junit5</artifactId>
-            <version>5.0.0-BETA-2</version>
+            <version>8b159a</version>
```
Comment (HardNorth) [inline example-junit5/pom.xml]: Guess that too outdated, please update dependencies.

File: example-junit/pom.xml [hunk 4]
```diff
@@ -14,9 +14,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.epam.reportportal</groupId>
+            <groupId>com.github.reportportal</groupId>
             <artifactId>agent-java-junit</artifactId>
-            <version>5.0.0-BETA-3</version>
+            <version>4ffd0d3d</version>
```
Comment (HardNorth) [inline example-junit/pom.xml]: Guess that too outdated, please update dependencies.

**Review by HardNorth — CHANGES_REQUESTED** [review]
What about Cucumbers and JBehave?

**Comment by HardNorth** [comment]
Do we still need this pull request? It's too outdated.

File: example-testng-logback/pom.xml (hunk 2 above)
Comment (Yumfriez) [reply example-testng-logback/pom.xml]: updated

File: example-junit5/pom.xml (hunk 3 above)
Comment (Yumfriez) [reply example-junit5/pom.xml]: updated

File: example-junit/pom.xml (hunk 4 above)
Comment (Yumfriez) [reply example-junit/pom.xml]: updated

File: example-testng-logback/src/main/resources/reportportal.properties (hunk 1 above)
Comment (Yumfriez) [reply example-testng-logback/src/main/resources/reportportal.properties]: reverted

**Comment by Yumfriez** [comment]
> Do we still need this pull request? It's too outdated.

Yes, we need examples of `Callback reporting functionality`. Branch updated

File: example-junit5/pom.xml [hunk 5]
```diff
@@ -14,10 +14,9 @@
 
     <dependencies>
         <dependency>
-            <!--            <groupId>com.epam.reportportal</groupId>-->
-            <groupId>com.github.reportportal</groupId>
+            <groupId>com.epam.reportportal</groupId>
             <artifactId>agent-java-junit5</artifactId>
-            <version>9e1592e</version>
+            <version>5.0.0-BETA-2</version>
```
Comment (HardNorth) [inline example-junit5/pom.xml]: BETA-2? It's BETA-9 already.

File: example-testng-logback/src/main/resources/reportportal.properties [hunk 6]
```diff
@@ -1,6 +1,7 @@
-rp.endpoint=http://example.com
-rp.api.key=api-key
-rp.launch=launch-name
-rp.project=project-name
-rp.tags=one;two;
+rp.endpoint=https://beta.demo.reportportal.io
+rp.uuid=51aabf1d-958c-4f14-92b5-00380884c3c5
+rp.launch=superadmin_TEST_EXAMPLE
+rp.project=reporting-test
+rp.attributes=key:value;value
```
Comment (HardNorth) [inline example-testng-logback/src/main/resources/reportportal.properties]: Please revert

**Review by HardNorth — CHANGES_REQUESTED** [review]

File: example-junit5/pom.xml (hunk 5 above)
Comment (Yumfriez) [reply example-junit5/pom.xml]: Updated to RC-1

File: example-testng-logback/src/main/resources/reportportal.properties (hunk 6 above)
Comment (Yumfriez) [reply example-testng-logback/src/main/resources/reportportal.properties]: fixed

**Review by HardNorth — APPROVED** [review]
